charts/traction/README.md

```diff
@@ -52,6 +52,12 @@ kubectl delete secret,pvc --selector "app.kubernetes.io/instance"=my-release
 
 ## Parameters
 
+| Name               | Description                                                                                                    | Value              |
+| ------------------ | -------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `nameOverride`     | String to override the helm chart name, second part of the prefix                                              | '""'               |
+| `fullnameOverride` | String to fully override the helm chart name, full prefix. ***Must be set when using a custom release name.*** | '""'               |
```

Comment:
Should we be very specific and say "a custom release name that does not include the workd traction"?

Reply:
good point